### ISSUE TITLE
Test failing error under Node 0.12

### ISSUE THREAD

**Issue by eddieajau** [balderdashy/sails-mysql#210]
Anyone else seeing this (Node v0.12.1)?:

```
{ [AssertionError: 2]
  name: 'AssertionError',
  actual: false,
  expected: true,
  operator: '==',
  message: 2,
  generatedMessage: false,
  uncaught: true }

  147 passing (2s)
  1 failing

1) Migratable Interface migrate: "alter" teardown and migrate existing data should retain the data when bootstrapped the second time:
     Uncaught 
  AssertionError: 2
      at /Users/martialeagle/git/eddieajau/sails-mysql/node_modules/waterline-adapter-tests/interfaces/migratable/migrate.alter.test.js:46:17
```

**Comment by devinivy** [comment]
Haven't checked this out yet.  @dmarcelino do the tests hit node 0.12?
Here's the failing assertion, that `1 == 2`: https://github.com/balderdashy/waterline-adapter-tests/blob/master/interfaces/migratable/migrate.alter.test.js#L46

**Comment by dmarcelino** [comment]
Pretty much every module/adapter (with few exceptions) is being tested against node 0.10 and 0.11. I'll check sails-mysql after lunch.

**Comment by dmarcelino** [comment]
sails-mysql is not presently being tested against 0.12:
https://github.com/balderdashy/sails-mysql/blob/master/.travis.yml#L2-L4

I've raised PR #214 to update that and the tests pass so I don't think node 0.12 is to blame for. Maybe this is caused by some sort of racing condition or a sync'ed callback that needs to be made async with  `process.nextTick()`?

**Comment by sailsbot** [comment]
Thanks for posting, @eddieajau.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
